File: problems/fib/fib.cpp
#include <iostream>

int fib(int n) {
  if (n <= 1)
    return n;
  return fib(n - 1) + fib(n - 2);
}

int main(int argc, char **argv) {
  int n = 34 + argc;
  int res = fib(n);
  if (res == 9227465)
    return 0;
  return 1;
}

File: problems/fib/fib.rs
fn fib(n: i32) -> i32 {
    if n <= 1 { 
        return n; 
    }
    fib(n - 1) + fib(n - 2)
}

fn main() {
    let args: Vec<String> = std::env::args().collect();
    let n = 39 + args.len() as i32;
    let res = fib(n);
    if res != 102334155 {
        std::process::exit(1);
    }
}
